benefit/data/sheets/AuthorisationCaseField/AuthorisationCaseField.json

```diff
@@ -676,6 +676,7 @@
   {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "CaseFieldID": "exceptionRecordReference", "UserRole": "caseworker-sscs-clerk", "CRUD": "R"},
   {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "CaseFieldID": "furtherEvidenceAction", "UserRole": "caseworker-sscs-clerk", "CRUD": "CRU"},
   {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "CaseFieldID": "hearings", "UserRole": "caseworker-sscs-clerk", "CRUD": "CRUD"},
+  {"LiveFrom": "04/03/2022", "CaseTypeID": "Benefit", "CaseFieldID": "hearingRoute", "UserRole": "caseworker-sscs-superuser", "CRUD": "CRUD"},
```

Comment:
Should check what the permissions need to be for this

Reply:
Updated now requires full CRUD permissions by all users